flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/rules/logical/FlinkFilterJoinRuleTest.java

```diff
@@ -240,6 +240,21 @@ public void testLeftJoinWithAllFiltersFromWhere() {
                 "SELECT * FROM MyTable1 LEFT JOIN MyTable2 ON true WHERE b1 = b2 AND c1 = c2 AND a2 = 2 AND b2 > 10 AND COALESCE(c1, c2) <> '' ");
     }
 
+    @Test
```

Comment:
please add some tests (including IT case) for inner joins

Reply:
Done!